### PR TITLE
actual outlined boxes

### PR DESCRIPTION
**Pull request by AscendedSion**
pleaes merge this

### CODE REVIEW

**Comment by canedoly** [comment]
you forgot about buildings and world esp boxes

**Comment by AscendedSion** [comment]
fart

File: Fedoraware/TeamFortress2/TeamFortress2/Features/ESP/ESP.cpp [hunk 1]
```diff
@@ -1248,6 +1249,7 @@ void CESP::DrawWorld() const
 
 					if (Vars::ESP::Main::Outlinedbar.Value)
 						g_Draw.OutlinedRect(x - 1, y - 1, w + 2, h + 2, Colors::OutlineESP);
+						g_Draw.OutlinedRect(x + 1, y + 1, w - 2, height - 2, Colors::OutlineESP);
```
Comment (lnx00) [inline Fedoraware/TeamFortress2/TeamFortress2/Features/ESP/ESP.cpp:1252]: This expression isn't covered by the if-statement in line 1164

**Comment by xxmyn** [comment]
just rewrite ESP :trollface:

**Comment by AscendedSion** [comment]
> just rewrite ESP :trollface:

true

File: Fedoraware/TeamFortress2/TeamFortress2/Features/ESP/ESP.cpp (hunk 1 above)
Comment (AscendedSion) [reply Fedoraware/TeamFortress2/TeamFortress2/Features/ESP/ESP.cpp:1252]: it works in getterhook so it works here

**Comment by hyperventilation** [comment]
when the sion is ascended

**Comment by AscendedSion** [comment]
> when the sion is ascended

when the ascendedsion is the sion ascendedsion ascended

**Comment by zphrus** [comment]
Wow 1 rectangle

**Comment by AscendedSion** [comment]
> Wow 1 rectangle

RAAAAAAAAAAAAAAAH

File: Fedoraware/TeamFortress2/TeamFortress2/Features/ESP/ESP.cpp (hunk 1 above)
Comment (lnx00) [reply Fedoraware/TeamFortress2/TeamFortress2/Features/ESP/ESP.cpp:1252]: Try disabling Outlinedbar. You will see that it still runs the second OutlinedRect call even though it's not intended.

**Pull request closed without merging**